src/plmlists.asm

```diff
@@ -414,6 +436,17 @@ if !AREA == 1
     org RoomState1RidleyMouth
     skip 20 : dw CustomPLMs_RidleyMouth
 endif
+
+; Ridley's Boss Room - Room $B32E
+org $8F8E98
+skip 4                  
+dw $805A                ; Set FRONT door as always solid grey so each version
+                        ; of the boss room is locked for true boss rando.
+skip 4
+dw $805B                ; Set BACK door as always solid grey so each version
+                        ; of the boss room is locked for true boss rando.
+dw $0000
```

Comment:
We can avoid the `dw $0000` because that's what is already in vanilla for those bytes.